options/rtdl/generic/linker.cpp

```diff
@@ -242,6 +242,8 @@ SharedObject *ObjectRepository::requestObjectWithName(frg::string_view name,
 
 		// TODO: Free this when the scope is no longer needed.
 		localScope = frg::construct<Scope>(getAllocator());
+	} else if (localScope == nullptr) {
```

Comment:
When will it be nullptr here?

Reply:
When called from `dlopen(..., RTLD_GLOBAL)` `localScope` is null and `createScope` is false. Same for `requestObjectAtPath`.

Reply:
Hmm... does it make sense to avoid passing `nullptr` in that case? Rather than enforcing it inside the function

Reply:
Actually, IG it'd be better to pass the global scope in from the outside, since that's what the other caller of `requestObjectWithName` does

Reply:
Yeah that's what I mean